prisma/schema.prisma

```diff
@@ -30,6 +30,9 @@ model User {
   apiKeyHash                String?              @unique(map: "users.apikeyhash_unique") @db.Char(64)
   apiKeyEnabled             Boolean              @default(false)
   tipDefault                Int                  @default(100)
+  tipRandom                 Boolean              @default(false)
+  tipRandomMin              Int?
+  tipRandomMax              Int?
```

Comment:
I don't think we need the boolean if we can just check for null on min and max. I'd suggest we use a postgres [range type](https://www.postgresql.org/docs/current/rangetypes.html) too but [prisma doesn't support it](https://github.com/prisma/prisma/issues/3287).